examples/bpi-monolith-test-suite/bpi/components/workgroup/workgroup.ts

```diff
@@ -6,16 +6,24 @@ export class Workgroup {
      name: string;
      participants: BpiSubject[] = [];
      worksteps: Workstep[] = [];
+     workstepids: Set<string> = new Set("");
 
     constructor(name: string, id: string, owner: BpiSubject, worksteps: Workstep[],) {
-        this.name = name;
         this.id = id;
+        this.name = name;
         this.worksteps = worksteps;
```

Comment:
since we are initializing `worksteps` here, should we also initialize `workstepIds` using `worksteps`? maybe simplest way for this would be
foreach workstep call addWorkstep?

Reply:
@Ybittan reminder about this comment :)